cmd/babylond/cmd/genhelpers/gentx.go

```diff
@@ -176,28 +176,27 @@ $ %s gentx my-key-name 1000000stake --home=/path/to/home/dir --keyring-backend=o
 			}
 
 			// bls load/create
-			privValKeyFilePath, statePath := PathsNodeCfg(clientCtx.HomeDir)
-			wrappedPV := privval.LoadWrappedFilePV(privValKeyFilePath, statePath)
-			if wrappedPV.Key.DelegatorAddress == "" {
-				cmd.PrintErrf("failed to read bls keys in %q. You probably forgot to run `babylond create-bls-key`, creating...\n", privValKeyFilePath)
-				wrappedPV, err = bls.CreateBlsKey(clientCtx.HomeDir, addr)
-				if err != nil {
-					return sdkerrors.Wrap(err, "failed to set bls keys into priv_validator_key.json")
-				}
-				cmd.PrintErrf("successfully created bls key in %q.\n", privValKeyFilePath)
+
+			ck, err := appsigner.LoadConsensusKey(clientCtx.HomeDir)
```

Comment:
Here we should basically check if the user already have the key setup, if it has there is no need to export, if there is no bls key set, export it

Reply:
so we should probably remove it here the call `appsigner.ExportGenBls(` 
@gitferry

Reply:
After offline discussing with @gitferry I understood 

`gen-bls-bbnvaloper1w60pd30puqlvma943fe5jcxsurlyfxzv3cxpvr.json` can be created only by `babylond gentx`

`bls_key.json` can be created by `bablyond init` or `babylond create-bls-key`

no need to remove `ExportGenBls` all good to merge